explorer/src/server.ts

```diff
@@ -3,16 +3,17 @@ import express from 'express'
 import helmet from 'helmet'
 import http from 'http'
 import mime from 'mime-types'
+import { Connection } from 'typeorm'
 import { Environment, ExplorerConfig } from './config'
 import * as controllers from './controllers'
 import { addRequestLogging, logger } from './logging'
 import adminAuth from './middleware/adminAuth'
 import seed from './seed'
 import { bootstrapRealtime } from './server/realtime'
 
-export default function server(conf: ExplorerConfig): Promise<http.Server> {
+export default function server(conf: ExplorerConfig, db: Connection): Promise<http.Server> {
```

Comment:
Why are we passing it around instead of just using `getConnection` within `seed`?